brayns/common/camera/Camera.cpp

```diff
@@ -24,11 +24,10 @@
 
 namespace brayns
 {
-Camera::Camera(const CameraType cameraType)
-    : _type(cameraType)
+Camera::Camera(const CameraType cameraType, const StereoMode stereoMode)
+    : _type{cameraType}
+    , _stereoMode{stereoMode}
```

Comment:
Not all combinations make sense for example stereo modes are nonsensical with orthographic projections. Throw for incorrect combinations?